src/compute-client/src/response.proto

```diff
@@ -69,7 +69,19 @@ message ProtoSubscribeBatch {
         int64 diff = 3;
     }
 
+    message ProtoSubscribeBatchContents {
+         oneof kind {
+             ProtoSubscribeUpdates updates = 1;
+             string error = 2;
+         }
+     }
+
+     message ProtoSubscribeUpdates {
+         repeated ProtoUpdate updates = 1;
+     }
+
     mz_repr.antichain.ProtoU64Antichain lower = 1;
     mz_repr.antichain.ProtoU64Antichain upper = 2;
-    repeated ProtoUpdate updates = 3;
```

Comment:
I don't think this change is not backwards-compatible. A backwards-compatible change will not remove
```
repeated ProtoUpdate updates = 3;
```
but will at first just rename the field name from `updates` to `updates_old`. The `RustType` implementation for `SubscribeUpdates` will then have to populate `self.updates` from `updates_old` or `updates` (currently it only uses the latter).